.travis.yml

```diff
@@ -12,6 +12,7 @@ install:
   - pip install .
 
 script:
+  - python setup.py install
```

Comment:
There is no point in this because we already do the `pip install .` as part of the install section.